README.md

```diff
@@ -69,60 +71,98 @@ This section explains how to install the latest stable release of
 Spyder. If you prefer testing the development version, please use
 the `bootstrap` script (see next section).
 
-The easiest way to install Spyder is:
+### The Easy/Recommended Way: Anaconda
 
-### On Windows:
+The easiest way to install Spyder for any of our platform, and
```

Comment:
for any ~~of our~~ platform

Reply:
Clarified and fixed to "any of our supported platforms". Obviously, there are plenty of platforms we/Anaconda don't support, but experts could conceivably get it running with one of the advanced methods.